README.md

```diff
@@ -3,23 +3,34 @@
 This is a powerful Todo list App to Manage tasks. Enjoy Watching!
 
```

Comment:
### Optional
- [ ] It would be better if we format our PR nicely, the current look and feel can be improved. Also, let our commit messages tell the coding story especially if the main task is adding and removing a todo

Reply:
Also, we are missing the icon in the input field, it would be nice to add it to achieve the given design